politeiawww/cmd/politeiaverify/README.md

```diff
@@ -0,0 +1,29 @@
+# Politeia Verify
+
+`politeiaverify` is a simple tool that allows anyone to independently verify 
+that Politeia has received your proposal/comment and that it is sound. The 
+input received in this command is the json bundle downloaded from the GUI.
+
+## Usage
+
+`politeiaverify [flags] <path to JSON bundle>`
+
+Flags:
```

Comment:
So I ended up not defining a default verification method for clarity sake. Do you agree?

I think this way it would be easier to add a verify invoice bundle with an additional flag, if we come to add that on cms gui.

Reply:
Can we make it so that the cli tool recognizes the filename and does the correct verifcation automatically?